src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,128 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { SearchParams } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink';
+
+const tableHeaders = ['Name', 'Sex', 'Born', 'Died'];
+
+type Props = {
+  people: Person[];
+  searchParams: URLSearchParams;
+  sortPeople: (
+    people: Person[],
+    sortKey: string | null,
+    order: string | null,
+  ) => Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  searchParams,
+  sortPeople,
+}) => {
+  const { personSlug = '' } = useParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || null;
+
+  const checkNameInPeople = useCallback((parentName: string | null) => {
+    if (!parentName) {
+      return '-';
+    }
+
+    const foundPerson = people.find(parent => parent.name === parentName);
+
+    return foundPerson ? <PersonLink person={foundPerson} /> : parentName;
+  }, [people]);
+
+  const sortedPeople = useMemo(() => {
+    if (!sort) {
+      return people;
+    }
+
+    return sortPeople(people, sort, order);
+  }, [sort, order, people]);
+
+  const getCorrectSearchParams: (tableHeader: string) =>
+  SearchParams = (tableHeader) => {
+    if (!sort || sort !== tableHeader) {
+      return { sort: tableHeader, order: null };
+    }
+
+    if (sort === tableHeader && !order) {
+      return { order: 'desc', sort: tableHeader };
+    }
+
+    return { sort: null, order: null };
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {tableHeaders.map(tableHeader => {
+            const sortedKey = tableHeader.toLowerCase();
+
+            return (
+              <th key={tableHeader}>
+                <span className="is-flex is-flex-wrap-nowrap">
+                  {tableHeader}
+                  <SearchLink
+                    params={getCorrectSearchParams(sortedKey)}
+                  >
+                    <span className="icon">
+                      <i className={classNames('fas',
+                        { 'fa-sort': !sort || sort !== sortedKey }, {
+                          'fa-sort-up': sort === sortedKey
+                          && order,
+                        }, {
+                          'fa-sort-down': sort === sortedKey
+                        && !order,
+                        })}
+                      />
+                    </span>
+                  </SearchLink>
+                </span>
+              </th>
+            );
+          })}
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {sortedPeople.map(person => {
+          const {
+            sex, born, died, fatherName, motherName, slug,
+          } = person;
```

Comment:
```suggestion
          const {
            sex, 
            born, 
            died, 
            fatherName, 
            motherName, 
            slug,
          } = person;
```
Provide a newline for each prop if their amount is too big